.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I don't think you meant to add this. You could add the path to gitignore instead

Reply:
Oops sorry about that. I'll fix the linter errors too.

Reply:
Oops, sorry. That should be removed now